Base.rte/Activities/OneManArmy.lua

```diff
@@ -55,7 +55,14 @@ function OneManArmy:StartActivity()
 		secondaryGroup = "Weapons - Secondary";
 		actorGroup = "Actors - Light";
 	end
-
+	-- Destroy all doors for this Activity
```

Comment:
If you don't open them before destroying them, their gibs will more likely block the paths when settling.

Reply:
I thought it might be that.
1. Do they open quick enough for that to help?
2. Would it be helpful to modify DestroyAllDoors in source so it can optionally delete them instead of gibbing them?
3. Alternative to 2, would it be helpful if it could optionally open them fully before gibbing them?

Reply:
They seem to spawn fully opened when doing the above, the result is exactly what I wanted.

Reply:
Okay cool, maybe it already does this. I'm overcomplicating stuff then, let's leave well enough alone and keep it as-is.